### ISSUE TITLE
Use `--project=@.` by default (again?)

### ISSUE THREAD

**Issue by goerz** [JuliaLang/IJulia.jl#1123]
It seems like when I run `IJulia.installkernel("Julia")`, the resulting kernel no longer has `"--project=@."`. It was the default in 
[v1.18](https://github.com/JuliaLang/IJulia.jl/releases/tag/v1.18.0), introduced in #820. I'm not seeing any release notes to indicate when the default changed.

There's https://github.com/JuliaLang/IJulia.jl/issues/928 about this. For the record, I strongly prefer the `--project=@.` behavior as a default. In any case, the issue is still open, and without the change being in a changelog, this seems like a "bug".

Can we go back to the old behavior, pretty please? ;-)

**Comment by goerz** [comment]
Or, at the very least, can we have a very prominent note in the README and the docs that explains the situation and recommends running

```
IJulia.installkernel("Julia", "--project=@.")
```

for those who like the behavior

**Comment by JamesWrigley** [comment]
I'm going to close this because I don't think that behaviour was ever the default; #820 only set that option in the build script and from looking through the git history it doesn't seem to have ever been the default argument to `installkernel()`.

**Comment by goerz** [comment]
Yes, sorry, I was confused. I only understood yesterday, after it was pointed on on Slack, that the "default" kernel I was referring to is the one defined in [`deps/build.jl`](https://github.com/JuliaLang/IJulia.jl/blob/5c6f79eefef64885c54be0623e0f8fcf8866469f/deps/build.jl#L6). That still has the `--project=@.` flag.

I don't actually understand exactly when the "build" happens: With `juliaup`, I'm regularly changing between different Julia versions, and I always have to install the `IJulia` package in each newly installed version of Julia. But that does not seem to automatically install or update the kernel. Hence my confusion. But anyway, I'm okay with creating kernels manually.

With my new and improved understanding, I don't think I'd still be asking for `IJulia.installkernel` to have "defaults". The PR in #1153 is the appropriate response to this.
